src/components/gui/gui.jsx

```diff
@@ -66,8 +66,10 @@ const GUIComponent = props => {
         backpackVisible,
         blocksTabVisible,
         cardsVisible,
+        canChangeLanguage,
```

Comment:
There are a lot of props that are all related to the menu bar. Would it make sense to refactor into a menuOptions object? E.g.:
```
menuOptions = {
    canChangeLanguage: true,
    canEditTitle: false,
    canManageFiles: true,
    canRemix: false,
    canShare: false,
    enableCommunity: false
}
```
Perhaps other options should also be in there (e.g. canCreateNew)

Reply:
I personally prefer the flat structure since it's less to keep in your head, and easier to define, override, etc.